ipaserver/install/bindinstance.py

```diff
@@ -990,11 +984,8 @@ def __setup_named_conf(self):
         named_txt = ipautil.template_file(
             os.path.join(paths.USR_SHARE_IPA_DIR, "bind.named.conf.template"),
             self.sub_dict)
-        named_fd = open(paths.NAMED_CONF, 'w')
-        named_fd.seek(0)
-        named_fd.truncate(0)
-        named_fd.write(named_txt)
-        named_fd.close()
+        with open(paths.NAMED_CONF, 'w') as named_fd:
+            named_fd.write(named_txt)
```

Comment:
```suggestion
        with open(paths.NAMED_CONF, 'w') as f:
            os.fchmod(f.fileno(), 0o600)
            os.fchown(f.fileno(), 0, self.named_gid)
            f.write(named_txt)
```

Reply:
600?

Reply:
Permissions are under the control of BIND package (platform-specific) and usually are `640,root,named`.
But to be sure they are correct we can explicitly set ones.

Reply:
Fixed.

Reply:
sorry, ``0o640`` is right